public/js/main.js

```diff
@@ -16,6 +16,9 @@ const selectors = require("./selectors");
 
 const App = require("./components/App");
 
+const { Services } = require("devtools-modules");
+Services._defaultPrefsEnabled = false;
```

Comment:
Don't we need this turned on though if we're in the Firefox panel, and this file still runs there?